src/OpenLoco/src/GameCommands/Company/RenameCompanyOwner.cpp

```diff
@@ -108,8 +110,79 @@ namespace OpenLoco::GameCommands
     }
 
     // 0x00434BA1
-    void sub_434BA1()
+    void tryLoadInstalledFaceForName(const CompanyId id, const char* companyName)
     {
-        call(0x00434BA1);
+        const auto competitorInstalledObjects = CompanyManager::findAllOtherInUseCompetitors(id);
+
+        std::optional<ObjectHeader> foundInstalledObject = std::nullopt;
+        for (const auto& object : ObjectManager::getAvailableObjects(ObjectType::competitor))
+        {
+            auto res = std::find(competitorInstalledObjects.begin(), competitorInstalledObjects.end(), object.first);
+            if (res != competitorInstalledObjects.end())
+            {
+                continue;
+            }
+
+            // Copy the string as it needs some processing
+            std::string objectName = object.second._name;
+            // Not sure what ControlCodes::pop16 is doing in the object name but it is at the start of all the object names
+            objectName.erase(std::remove(std::begin(objectName), std::end(objectName), ControlCodes::pop16));
+
+            auto strcmpSpecial = [](const char* lhs, const char* rhs) {
+                while (*lhs && *rhs)
+                {
+                    const auto chr = *lhs;
+                    if (chr == '\xFF')
```

Comment:
This is interesting in that this is clearly some sort of control code but it isn't a valid one for drawing. So its all dead code. Mysterious...

Reply:
Guessing this could be related to handling multibyte characters? Not active in the English version, but it wouldn't be the first remnant either.

Reply:
If so it's a very odd multibyte format.